src/extends/room/meta.js

```diff
@@ -164,6 +164,12 @@ Room.isHallway = function (name) {
 }
 
 Room.isClaimable = function (name) {
+  if (Game.gcl.level >= GCL_NOVICE && Game.map.getRoomStatus(name).status !== 'normal') {
```

Comment:
What is this condition for?

Reply:
This will be updated to >= GCL_NOVICE && Game.map.getRoomStatus(name).status === 'novice' in order to check if the room can be claimed. Like the above comment suggested.